src/components/Dashboard.vue

```diff
@@ -92,6 +93,31 @@
           </p>
         </InfoButton>
       </div>
+      <div
+          class="
+          flex
+          ml-3
+          md:ml-0
+          justify-start
+          items-center
+          mt-6
+          sm:space-x-2
+          md:space-x-4
+        "
+      >
+        <SwitchButton
+            label="Are you within your first 24 months of starting your activity?"
+            v-model="secondYear"
+            :key="secondYearKey"
+        />
+        <InfoButton
+            link="https://www.cgd.pt/Site/Saldo-Positivo/leis-e-impostos/Pages/irs-trabalhadores-independentes.aspx"
+        >
+          <p class="text-sm w-64 text-center">
+            You get a 50% discount on your IRS tax in your first 12 months, and a 25% discount on your first 24 months. Click to see more.
```

Comment:
@bolt12 good point! It is actually very confusing. Maybe I send a message to the AT via e-balcão to see what they say?

Reply:
Sounds reasonable to me.
Let me do that now!

Reply:
also, after this I think we are all eligible to be certified accountants

Reply:
hahaha. true that!
Anyway, I spotted one thing with the Social Security reductions 5%, 10% ... 25%. When I try to use 5% or 10% reductions I get no change. Is it normal (because of a possible high income)?

Reply:
Thank you for fetching that up @franciscobmacedo, couldn't put it in any better way ! That's exactly what I was looking for! ( I am saving your comment for future reference )

> also, after this I think we are all eligible to be certified accountants

The saddest thing is that this is true, I haven't had much luck with accountants regarding my fiscal situation. Also the law could be more clear and its implementation should be open-source! Public money, public code. This would save us a lot of trouble 😄 

You won't imagine my headache with last years IRS submission with IRS Jovem and discounted coefficients. Both were documented but there was no specification for how they interacted with each other... does one overwrite the other, does it accumulate? Only god knows...

Reply:
It is indeed a pain point in the country.

@pedrosimao good point. I believe the SS discount is currently only being applied if it's not the first and last year. According to that [document](https://www.seg-social.pt/documents/10152/15974914/1009%20Trabalhador%20independente%20-%20novo%20regime/87b6e00c-523d-4718-8a88-942ea804c18a):

> No momento da declaração trimestral, os Trabalhadores Independentes, podem optar pela fixação de
um rendimento superior ou inferior até 25%, esta opção é efetuada em intervalos de 5% (5%, 10%,
15%, 20%, 25%) àquele que resultar dos valores declarados, sem prejuízo dos limites previstos [mínimo
de 20,00€ e máximo de 12xIAS (5.765,16€ valor em 2023)

I'm not sure what "sem prejuizo dos limites previstos" means.... But I think it makes sense that the discount is aplicable to the the 12xIAS value as well - so that needs to be fixed as well.

Reply:
Well, I feel like there is a mistake here.
I think the way it should be calculated is that the SS tax is applied to a maximum of 5.765,16 euros.
Let me try to fix it here already.

Reply:
the getter `ssPay` is doing that: multiplying `ssTax` with the minimum between `SS_MAX_MONTH_INCOME` (5.765,16) and 70% of gross income (with or without discounts). I think the fix that needs to be done is to apply the discounts to `SS_MAX_MONTH_INCOME` as well.

Reply:
I also think that maybe yearSS should just be monthSS*12

Reply:
Ok I also fixed it inside the same PR. The problem is that the discount was being done inside the Math.min calculation. I now can get the same results as on this calculator here:
https://www.doutorfinancas.pt/calculadora-seguranca-social-trabalhadores-independentes-2019/

Reply:
perfect, that's it!